### PR TITLE
taint fix and more

### PR DESCRIPTION
**Pull request by vanhauser-thc**
this:
 
  * fix for taint (a little step forward)
  * update the benchmark documentation
  * combine the two benchmark runs

### CODE REVIEW

**Comment by vanhauser-thc** [comment]
@lszekeres is has arrived :)

File: docs/reference/benchmarks.md [hunk 1]
```diff
@@ -12,31 +12,33 @@ This page describes each benchmark that is part of fuzzbench.
...
-| re2-2014-12-09              | fuzz-target            | False           | 0                     | 6547                     | 6.02                 |
-| sqlite3_ossfuzz             | ossfuzz                | True            | 1258                  | 45136                    | 7.9                  |
-| systemd_fuzz-link-parser    | fuzz-link-parser       | False           | 6                     | 53453                    | 5.91                 |
-| vorbis-2017-12-11           | fuzz-target            | False           | 1                     | 5022                     | 6.0                  |
-| woff2-2016-05-06            | fuzz-target            | False           | 62                    | 10923                    | 6.72                 |
-| zlib_zlib_uncompress_fuzzer | zlib_uncompress_fuzzer | False           | 0                     | 875                      | 5.69                 |
+| Benchmark                   | Fuzz target            | Has dictionary? | Is stable? | Seed type\*              | Number of seed inputs  | Number of progam edges\*\*| Binary Size (MB)\*\*\*|
```
Comment (jonathanmetzman) [inline docs/reference/benchmarks.md:15]: ehhh...I'd rather have a percentage if possible but this is a good start.

File: docs/reference/benchmarks.md [hunk 2]
```diff
@@ -12,31 +12,33 @@ This page describes each benchmark that is part of fuzzbench.
...
+| lcms-2017-03-21             | fuzz-target            | False           | True       | ICC profile              | 1                      | 6959                      | 6.11                  |
+| libjpeg-turbo-07-2017       | fuzz-target            | False           | False      | JPEG                     | 1                      | 9586                      | 6.4                   |
+| libpcap_fuzz_both           | fuzz_both              | False           | False      | PCAP                     | 0                      | 8149                      | 6.14                  |
+| libpng-1.2.56               | fuzz-target            | False           | False      | PNG                      | 1                      | 2991                      | 5.8                   |
+| libxml2-v2.9.2              | fuzz-target            | False           | False      | XML                      | 0                      | 50461                     | 8.23                  |
+| mbedtls_fuzz_dtlsclient     | fuzz_dtlsclient        | False           | True       | special                  | 1                      | 10942                     | 6.67                  |
```
Comment (jonathanmetzman) [inline docs/reference/benchmarks.md:27]: maybe "custom" instead of special

File: docs/reference/benchmarks.md (hunk 1 above)
Comment (inferno-chromium) [reply docs/reference/benchmarks.md:15]: +1, can either stable column or add percents, i dont think all other targets are fully unstable.

**Comment by inferno-chromium** [comment]
Lets use this CL for just the doc change since we need to start experiment and doc change can wait. we need stability percent and that slight custom->special nit.

File: docs/reference/benchmarks.md (hunk 1 above)
Comment (vanhauser-thc) [reply docs/reference/benchmarks.md:15]: well anything less than 100% is unstable, and 14 benchmarks are.
I ran the experiments for ~15 minutes and usually the stability increases over time (instability is often hash maps or threads and they happen with most fuzz attempts) as the coverage increases over time, increasing the number of stable map entries vs the unstable ones.
so if you want to have numbers you need them from a full experiment. you can find them in any fuzzer_stats file of any afl variant of an already run experiment.
And I must say I deleted my data. I dont build in the cloud but on my machine and I do not have endless disk space... plus the building takes ages and hogs my cpu ... :)

File: service/experiment-requests.yaml [hunk 3]
```diff
@@ -21,6 +21,10 @@
 
 - experiment: 2020-08-13
   fuzzers:
+    - libfuzzer_fixcrossover
```
Comment (inferno-chromium) [inline service/experiment-requests.yaml:24]: This experiment has started, so this change can be skipped.

File: docs/reference/benchmarks.md (hunk 1 above)
Comment (inferno-chromium) [reply docs/reference/benchmarks.md:15]: Stability column, we will validate and add after 2/3 weeks. Can you please remove it from this PR.

File: docs/reference/benchmarks.md [hunk 4]
```diff
@@ -12,31 +12,33 @@ This page describes each benchmark that is part of fuzzbench.
...
-| re2-2014-12-09              | fuzz-target            | False           | 0                     | 6547                     | 6.02                 |
-| sqlite3_ossfuzz             | ossfuzz                | True            | 1258                  | 45136                    | 7.9                  |
-| systemd_fuzz-link-parser    | fuzz-link-parser       | False           | 6                     | 53453                    | 5.91                 |
-| vorbis-2017-12-11           | fuzz-target            | False           | 1                     | 5022                     | 6.0                  |
-| woff2-2016-05-06            | fuzz-target            | False           | 62                    | 10923                    | 6.72                 |
-| zlib_zlib_uncompress_fuzzer | zlib_uncompress_fuzzer | False           | 0                     | 875                      | 5.69                 |
+| Benchmark                   | Fuzz target            | Has dictionary? | Seed type\*              | Number of seed inputs  | Number of progam edges\*\*| Binary Size (MB)\*\*\*|
```
Comment (inferno-chromium) [inline docs/reference/benchmarks.md:15]: Would you be ok to fix the "Has dictionary column", i see more 

```
jsoncpp_jsoncpp_fuzzer/build.sh:cp $SRC/jsoncpp/src/test_lib_json/fuzz.dict $OUT/jsoncpp_fuzzer.dict
lcms-2017-03-21/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
lcms-2017-03-21/Dockerfile:    https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/icc.dict
libpng-1.2.56/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
libpng-1.2.56/Dockerfile:    https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/png.dict
libxml2-v2.9.2/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
libxml2-v2.9.2/Dockerfile:    https://raw.githubusercontent.com/google/AFL/debe27037b9444bbf090a0ffbd5d24889bb887ae/dictionaries/xml.dict
openssl_x509/build.sh:cp fuzz/oids.txt $OUT/x509.dict
php_php-fuzz-parser/build.sh:cp sapi/fuzzer/dict/parser $OUT/php-fuzz-parser.dict
proj4-2017-08-14/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
proj4-2017-08-14/Dockerfile:    https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/proj4.dict
re2-2014-12-09/Dockerfile:RUN wget -qO $OUT/fuzz-target.dict \
re2-2014-12-09/Dockerfile:    https://raw.githubusercontent.com/google/fuzzing/master/dictionaries/regexp.dict
sqlite3_ossfuzz/Dockerfile:COPY build.sh *.dict $SRC/
sqlite3_ossfuzz/build.sh:cp $SRC/*.dict $SRC/*.zip $OUT/
```

Comment (vanhauser-thc) [reply docs/reference/benchmarks.md:15]: hadnt touched it but ok ... ;)

File: service/experiment-requests.yaml (hunk 3 above)
Comment (vanhauser-thc) [reply service/experiment-requests.yaml:31]: yup done

File: docs/reference/benchmarks.md (hunk 4 above)
Comment (inferno-chromium) [reply docs/reference/benchmarks.md:15]: :) got you!

**Pull request merged**